Virtual Librarian/VirtualLibrarian.API/Core/Library.cs

```diff
@@ -60,6 +60,19 @@ public ICollection<User> GetAllUsers()
             return _readerWriter.GetUsers();
         }
 
+        public int GetUserId(string userName)
+        {
+            ICollection<User> allUsers = new List<User>();
```

Comment:
you should probably use the DBReaderWriter method GetUsers to get all users and then do smth with them.